### ISSUE TITLE
Module Text with Image: Insert character limit

### ISSUE THREAD

**Issue by hmkay** [liqd/digitalstrategie#320]
**URL:** https://digitalstrategie-dev.liqd.net/
**user:** participant
**expected behaviour:** there is only as much text possible to insert as fits next to the picture on desktop version
**behaviour:** one can insert a lot of text.
**important screensize:** desktop, mobile
**device & browser:** MacOS, Firefox

![Uploading Bildschirmfoto 2022-03-23 um 16.47.48.png…]()

**Comment by hmkay** [comment]
Hm, maybe "only as much text as fits next to picture" will be hard in cases of a smaller picture

<img width="1059" alt="Bildschirmfoto 2022-03-23 um 16 56 48" src="https://user-images.githubusercontent.com/30862422/159742150-51667aba-c05b-4e23-a204-a96b013237ef.png">

So probably just a char limit and aligning the text to the right side would be the way to go...

**Comment by phillimorland** [comment]
@hmkay @maxliqd so we use richtext here and that cannot have a char limit because richtext uses html tags for formatting and they would be counted as characters (its a long standing internet issue) anyway would you be ok for this to just be a text field without the ability to add lists ect.? we can then add a char limit, also what should limit be?

**Comment by hmkay** [comment]
That sounds good, @phillimorland  - we don´t need any lists etc. Is it possible to have italic or bold text? 
And about the char limit, I dunno. What would you suggest would suit good?

**Comment by phillimorland** [comment]
@hmkay no there will be no bold ect. no idea on limit, don't know what it will be used for, if you also don't know you can test and see what fits with the image

**Comment by hmkay** [comment]
I think it will be about 700-900 characters, so I guess we can put 900? Or 1000? And then we test it?
Also we do need a headline - I dunno if that makes it more complicated, @phillimorland ?

**Comment by phillimorland** [comment]
@hmkay in addition to the title?

**Comment by hmkay** [comment]
Nope! That would be the headline, sorry, @phillimorland , forgot that the title was independent from the text field!

**Comment by fuzzylogic2000** [comment]
@hmkay What is to be done here? 

- [ ]  Change the richtextfield to a normal textfield (no bold, no underline, no lists, no headlines, ...) 
- [ ] add a character limit of 1000? (If you are unsure about the limit you can also test yourself, by counting the characters next to your images. (In an office program that does the counting for you.)))
- [ ] The title thing is not valid anymore, as there is a title, we do not need to another one, is that right?

**Comment by hmkay** [comment]
Hello @fuzzylogic2000 , yes, let´s do normal textfield, 800 char limit, and no headline in text field (as there is already the title). Thaaaanks!!! =)
